src/logger.ts

```diff
@@ -1,118 +1,36 @@
-import { normalizeValue } from "./utils"
+import { Logger, LoggerOptions } from "opstooling-js"
 
-// TODO: replace this module with logger from opstooling-js
-
-type LoggingImplementation = {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  log: (...args: any[]) => void
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  error: (...args: any[]) => void
-}
-
-enum LoggingLevel {
-  info,
-  warn,
-  error,
-  fatal,
-}
-type LoggingLevels = keyof typeof LoggingLevel
-
-export class Logger {
-  constructor(
-    public options: {
-      name: string
-      logFormat: "json" | null
-      minLogLevel: LoggingLevels
-      impl: LoggingImplementation
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      context?: Record<string, any>
-    },
-  ) {}
-
-  child(context: Record<string, unknown>): Logger {
-    /*
-      Adjust keys in order to prevent overriding existing data in the context
-      with the same key
-    */
-    const currentContextKeys = Object.keys(this.options.context ?? {})
-    const adjustedContext: { [key: string]: unknown } = {}
-    for (const [key, value] of Object.entries(context)) {
-      let suggestedName = key
-      while (currentContextKeys.includes(suggestedName)) {
-        suggestedName = `${suggestedName} (${new Date().toISOString()})`
-      }
-      adjustedContext[suggestedName] = value
-      currentContextKeys.push(suggestedName)
+export const logFormat = ((): "json" | null => {
+  const value = process.env.LOG_FORMAT
+  switch (value) {
+    case "json": {
+      return value
+    }
+    case undefined: {
+      return null
+    }
+    default: {
+      throw new Error(`Invalid $LOG_FORMAT: ${value}`)
```

Comment:
Shouldn't this belong in `opstooling-js` too?

Reply:
it should, but I decided to not do too much of unrelated changes, as there is another ticket for opstooling-js